molo/polls/test_templates/base.html

```diff
@@ -29,10 +29,6 @@
     <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    {% compress css %}
```

Comment:
How come we are removing this?

Reply:
I can't see that it's needed and it breaks the build. There's a commit in molo 5.something which removes this file so the compress complains that it can't find it.

Reply:
cool